lib/config/fileconf_test.go

```diff
@@ -544,21 +544,7 @@ func TestX11Config(t *testing.T) {
 				DisplayOffset: x11.DefaultDisplayOffset,
 				MaxDisplay:    100,
 			},
-		}, {
-			// DELETE IN 10.0.0 (Joerger): yaml typo, use max_display.
-			desc: "max displays set",
-			mutate: func(cfg cfgMap) {
-				cfg["ssh_service"].(cfgMap)["x11"] = cfgMap{
-					"enabled":      "yes",
-					"max_displays": 100,
-				}
-			},
-			expectX11Config: &x11.ServerConfig{
-				Enabled:       true,
-				DisplayOffset: x11.DefaultDisplayOffset,
-				MaxDisplay:    100,
-			},
-		}, {
+		}, {}, {
```

Comment:
Do we need to keep the empty test block?

```suggestion
		}, {
```